app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java

```diff
@@ -98,6 +102,22 @@ public static synchronized DatabaseHelper getInstance(Context context) {
         return dbInstance;
     }
 
+    /**
+     * returns test DatabaseHelper instance. If the instance already exists it returns exiting instance, if not, it
+     * creates a instance and returns it. This insures there is only one instance of test DatabaseHelper for the
+     * application.
+     *
+     * @param context an activity's context
+     * @return an instance of the test DatabaseHelper
+     **/
+    public static synchronized DatabaseHelper getTestInstance(Context context) {
```

Comment:
In terms of how this new database helper function address the issue of "potentially deleting all data in the database",  Does this function mitigate this problem by creating a new database?

Reply:
yes this function is to get the separate instance of the test database, so it won't interfere with app database.